lib/messenger_platform/callback_registry.rb

```diff
@@ -11,14 +11,8 @@ def register(klass)
         @callbacks[klass.new.callback_name] = klass.to_s
       end
 
-      def process_request(raw)
-        json = JSON.parse(raw)
-        CallbackParser.new(json.deep_dup).parse do |event|
-          @callbacks[event.webhook_name].constantize.new.run(event, json.deep_dup) if @callbacks[event.webhook_name]
-        end
-        true
-      rescue JSON::ParserError => e
-        false
+      def get_callback_handler_for(webhook_name)
```

Comment:
Since this lives in the `CallbackRegistry`, even `callback` could arguably be redundant.